c2corg_ui/templates/outing/index.html

```diff
@@ -27,7 +27,7 @@
   <div class="documents-nav-buttons text-center">
     <button class="btn btn-default show-documents-filters-phone" translate>Filters</button>
     ${add_doctype_selector()}
-    <button class="btn btn-default" protected-url-btn url="${request.route_path('outings_add')}">
+    <button class="btn btn-default orange-btn" protected-url-btn url="${request.route_path('outings_add')}">
```

Comment:
tooltip ok, but it won't work well on smartphone and adding text takes space, and we are already short on space. I don't really know what would be the solution :P